modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,290 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a hosted monitoring and log storage service. You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
+
+To configure log forwarding to CloudWatch, create a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch and a pipeline that uses the output.
+
+You can include a specific CloudWatch topic in the output or use the default. The CloudWatch output can use a TCP (insecure) or TLS (secure TCP) connection.
```

Comment:
>The CloudWatch output can use a TCP (insecure) or TLS (secure TCP) connection.
It's HTTPS, but this part is hidden under the hood, we don't expose it.

Reply:
Updated. I removed line 12 altogether. Thanks.